text/0000-yarn-create.md

```diff
@@ -0,0 +1,67 @@
+- Start Date: 2017-03-31
+- RFC PR: (leave this empty)
+- Yarn Issue: (leave this empty)
+
+# Summary
+
+The idea would be to make it easier for users to setup projects from scratch.
+
+# Motivation
+
+During the past few years, we've seen an increase of the number of "boilerplate" projects, each one aiming to lower the complexity of creating new projects with Javascript. Create React App is a good example, but we can also mention Neutrino, released by the Mozilla teams, or Next.js, which recently reached its v2 milestone.
+
+Despite their utility, using these tools still require to manually install them, and then to keep them updated. A common workaround has been to prone the use of a "core" package to be installed locally, and a "cli" package to be installed globally that would act as a bridge to the core package. It feels like a hack, and maybe we could do something to help both project maintainers their users.
+
+# Detailed design
+
+This RFC suggests to add a new command:
+
+```
+$> yarn create <package-name>
+```
+
+Running this command would have the samne effect as:
+
+```
+$> yarn add --dev <package-name>
```

Comment:
Maybe not, cf the third point https://github.com/arcanis/rfcs/blob/c6afb08a2a516c5dca4d03429c31bd8f91a9a03f/text/0000-yarn-create.md#alternatives